ts/defs.bzl

```diff
@@ -163,24 +163,28 @@ def ts_project(
 
         out_dir: String specifying a subdirectory under the bazel-out folder where outputs are written.
             Equivalent to the TypeScript --outDir option.
+
             Note that Bazel always requires outputs be written under a subdirectory matching the input package,
-            so if your rule appears in path/to/my/package/BUILD.bazel and out_dir = "foo" then the .js files
-            will appear in bazel-out/[arch]/bin/path/to/my/package/foo/*.js.
-            By default the out_dir is '.', meaning the packages folder in bazel-out.
+            so if your rule appears in `path/to/my/package/BUILD.bazel` and out_dir = "foo" then the .js files
+            will appear in `bazel-out/[arch]/bin/path/to/my/package/foo/*.js`.
+
+            By default the out_dir is the package's folder under bazel-out.
 
         tsc: Label of the TypeScript compiler binary to run.
             This allows you to use a custom compiler.
         tsc_worker: Label of a custom TypeScript compiler binary which understands Bazel's persistent worker protocol.
         validator: Label of the tsconfig validator to run when `validate = True`.
         allow_js: Whether TypeScript will read .js and .jsx files.
             When used with `declaration`, TypeScript will generate `.d.ts` files from `.js` files.
-        resolve_json_module: None | Boolean; Specifies whether TypeScript will read .json files. Defaults to None.
```

Comment:
This makes me wonder if this param is actually required? 🤔 

It's essentially only used for determining the default `srcs`...

Reply:
It's not required, default is None

Reply:
I mean required at all. Can we just delete the param and code for it? The default `srcs` can not longer use it, but that's just the defaults and imo not worth a param like this...